extensions/amp-base-carousel/1.0/base-carousel.jss.js

```diff
@@ -85,6 +85,11 @@ const slideElement = {
   justifyContent: 'center',
   scrollSnapAlign: 'start',
   scrollSnapStop: 'always',
+  '[snap="false"] > &': {
```

Comment:
IMHO this should be a class set on the scroller (or maybe even each slide). I'd like to reduce the reliance on non-standard attributes for CSS.

Reply:
Done